src/countries/vn.js

```diff
@@ -14,7 +14,7 @@ var fixVnShortPhone = tools.fixPhoneBuilder(8, 11, config);
 var fixVnLongPhone = tools.fixPhoneBuilder(9, 12, config);
 
 var fixVnPhone = function (phone) {
-  phone = phone.replace(/[^\d\+]/g, '').replace(/^00/g, '')
+  phone = phone.replace(/[^\d\+]/g, '').replace(/^00/g, '').replace(/^0/g, '')
```

Comment:
`.replace(/^00/g, '').replace(/^0/g, '')` could be simplified
at least one option is `.replace(/^0{1,2}/g, '')`